types/filenamify/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
```

Comment:
I don't see anything which would trigger this rule in the `.d.ts` file - why is this here? Can this comment (and the entire `rules` section containing it) be removed?